includes/notes/class-wc-stripe-upe-availability-note.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Display a notice to merchants to inform about UPE.
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use Automattic\WooCommerce\Admin\Notes\NoteTraits;
+use Automattic\WooCommerce\Admin\Notes\Note;
+use Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WC_Stripe_UPE_Availability_Note
+ */
+class WC_Stripe_UPE_Availability_Note {
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-stripe-upe-availability-note';
+
+	/**
+	 * Link to enable the UPE in store.
+	 */
+	const ENABLE_IN_STORE_LINK = '?TODO';
```

Comment:
We added a blank page for the onboarding page [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1737). You can use `wp-admin/admin.php?page=wc_stripe-onboarding_wizard` here.